frontend/components/commons/header/filters/SearchBar.vue

```diff
@@ -99,22 +99,30 @@ export default {
     max-width: 280px;
     margin-right: auto;
     margin-left: 0;
+    .--extended & {
+      min-width: 100%;
+    }
+  }
+  &__button {
+    cursor: pointer;
+    padding: 5px;
+    border-radius: 3px;
+    background: $lighter-color;
+    transition: background 0.2s ease-in-out;
+    &:hover {
+      transition: background 0.2s ease-in-out;
+      background: palette(grey, bg)
+    }
```

Comment:
If these styles are related to the new rules management view shold not be in a common component.

If is just a style refactoring, should be in a separate PR. We have already a lot changes in this PR to review.

Reply:
This is a new style created for the rule search bar, but it affects other